src/test/resources/detekt-config.yml

```diff
@@ -19,7 +19,7 @@ complexity:
     threshold: 30
   LongParameterList:
     active: true
-    functionThreshold: 4
+    functionThreshold: 5
```

Comment:
what code is violating this?

Reply:
ChangeLevelCalculator.calculate()

Reply:
how come Detekt passes in main? why make this change?